ui/custom.d.ts

```diff
@@ -0,0 +1,19 @@
+declare module '*.svg' {
+  const content: any
+  export default content
+}
+
+declare module '*.gif' {
+  const content: any
+  export default content
+}
+
+declare module '*.css' {
+  const classes: { [key: string]: string }
```

Comment:
because the  type is just `{ [key: string]: string }` ,seems something like this is not enable :
```javascript
import { innerProp }  from 'xxx.module.scss'
```
 just work for whole module importing:
```javascript
import styles  from 'xxx.module.scss'
```